rules/__tests__/prefer-todo.test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { RuleTester } = require('eslint');
+const rule = require('../prefer-todo');
+
+const ruleTester = new RuleTester({
+  parserOptions: { ecmaVersion: 2015 },
+});
+
+ruleTester.run('prefer-todo', rule, {
+  valid: [
+    'test.todo("i need to write this test");',
+    'test("stub", () => expect(1).toBe(1));',
```

Comment:
https://github.com/facebook/jest/blob/665a93eec70c4f2d7809d1848fbc8ca60cfd8361/e2e/test-in-root/spec.js#L9